javascript2.knockout/licenseinfo.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+<licenseinfo>
+    <fileset>
+        <file>src/org/netbeans/modules/javascript2/knockout/model/resources/knockout-2.2.1.model</file>
+        <file>src/org/netbeans/modules/javascript2/knockout/model/resources/knockout-3.0.0.model</file>
+        <file>src/org/netbeans/modules/javascript2/knockout/model/resources/knockout-3.2.0.model</file>
+    </fileset>
+    <license ref="Apache-2.0-ASF" />
```

Comment:
license + comment need to be children of fileset

Reply:
eh, silly copy/paste typo. Thanks for the hint.